gnome-image-installer/pages/install/gis-scribe.c

```diff
@@ -1110,6 +1110,8 @@ gis_scribe_tee_data_free (GisScribeTeeData *data)
   g_slice_free (GisScribeTeeData, data);
 }
 
+G_STATIC_ASSERT (G_MAXSSIZE <= G_MAXUINT64);
```

Comment:
What does this relate to? It’s not immediately obvious (from the diff view in GitHub) where you’re sticking a `gsize` value in a `guint64`.

⇒ Maybe add a comment

Reply:
It's in the function below but I think this check is (a) unnecessary (our images are certainly < 2 ** 62 - 1) or (b) incomplete (the gsize is repeatedly added to the guint64 so if I were really over-egging it I should add an overflow check). OK.

Reply:
Yeah, if you’re repeatedly adding the `gsize` to the `guint64` then it needs a runtime overflow check, if you care about that kind of thing.